sys-devel/llvm/llvm18-18.1.2.recipe

```diff
@@ -0,0 +1,792 @@
+	cmd:wasm_ld = $portVersion
+	devel:liblldCOFF$secondaryArchSuffix = $portVersion
+	devel:liblldCommon$secondaryArchSuffix = $portVersion
+	devel:liblldELF$secondaryArchSuffix = $portVersion
+	devel:liblldMinGW$secondaryArchSuffix = $portVersion
+	devel:liblldWasm$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_lld="
+	haiku$secondaryArchSuffix
+	lib:libLLVM$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_libs="
+	llvm18${secondaryArchSuffix}_libs = $portVersion
+	lib:libLLVM$secondaryArchSuffix = $portVersionCompat
+	lib:libLTO$secondaryArchSuffix = $portVersionCompat
+	lib:libRemarks$secondaryArchSuffix = $portVersionCompat
+	"
+REQUIRES_libs="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PYTHON3_VERSION="3.10"
+PROVIDES_python310="
+	llvm18${secondaryArchSuffix}_python310 = $portVersion
+	cmd:lit
+	"
+REQUIRES_python310="
+	setuptools_python310
+	cmd:python$PYTHON3_VERSION
+	"
+
+libunwindSoVersion="1"
+libunwindLibVersion="1.0"
+SUMMARY_libunwind="LLVM Unwinder - libunwind"
+DESCRIPTION_libunwind="libunwind is an implementation of the interface defined \
+by the HP libunwind project. It was contributed by Apple as a way to enable \
+clang++ to port to platforms that do not have a system unwinder. It is \
+intended to be a small and fast implementation of the ABI, leaving off some \
+features of HP's libunwind that never materialized (e.g. remote unwinding).
+
+The unwinder has two levels of API. The high level APIs are the _Unwind_* \
+functions which implement functionality required by __cxa_* exception \
+functions. The low level APIs are the unw_* functions which are an \
+interface defined by the old HP libunwind project."
+PROVIDES_libunwind="
+	llvm18${secondaryArchSuffix}_libunwind = $portVersion
+	lib:libunwind$secondaryArchSuffix = $libunwindLibVersion compat >= $libunwindSoVersion
+	"
+REQUIRES_libunwind="
+	haiku$secondaryArchSuffix
+	"
+
+SUMMARY_libunwind_devel="Headers and static libraries for LLVM Unwinder - libunwind"
+DESCRIPTION_libunwind_devel="libunwind is an implementation of the interface \
+defined by the HP libunwind project. It was contributed by Apple as a way to \
+enable clang++ to port to platforms that do not have a system unwinder. It is \
+intended to be a small and fast implementation of the ABI, leaving off some \
+features of HP's libunwind that never materialized (e.g. remote unwinding).
+
+The unwinder has two levels of API. The high level APIs are the _Unwind_* \
+functions which implement functionality required by __cxa_* exception \
+functions. The low level APIs are the unw_* functions which are an \
+interface defined by the old HP libunwind project."
+PROVIDES_libunwind_devel="
+	llvm18${secondaryArchSuffix}_libunwind_devel = $portVersion
+	devel:libunwind$secondaryArchSuffix = $libunwindLibVersion compat >= $libunwindSoVersion
+	"
+REQUIRES_libunwind_devel="
+	llvm18${secondaryArchSuffix}_libunwind == $portVersion base
+	haiku${secondaryArchSuffix}_devel
+	"
```

Comment:
Shouldn't there be a CONFLICTS here? Looking at the contect of the libunwind packages for LLVM17 and LLVM18 I  see both have the same files?

![llvm](https://github.com/haikuports/haikuports/assets/16057090/ade11dc5-289e-4cda-b5a4-b1cf52bcdd1a)